### DISCUSSION TITLE
The connection to Gridtracker suddenly broke.

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#1782] [General]
I don't understend situation, please help!
Gridtracker connection was configured and worked. After blackout on my server Cloudlog is operational. CloudCAT connects to Cloudlog without errors.
Gridtracker reports on click to "Test" button in the Gridtracker's logging settings "No Responce or Timeout" after approximately 10s of waiting. Nothing was changed, API key is operational. If i would open url "https://r1blh.cats-home.net/log/index.php/api/qso" i will get message:
```
"
status | "failed"
reason | "wrong JSON"
"

```

How to solve this situation? Clean reinstallation of the Gridtracker gave nothing.

**Comment by phl0** [comment]
It seems Gridtracker cannot talk to Cloudlog any more. Could you please check the logs for an requests made by Gridtracker?

> Nothing was changed, API key is operational. If i would open url "https://r1blh.cats-home.net/log/index.php/api/qso" i will get message:

Thats normal because you didn't supply any QSO data with your request in the browser.

**Reply by R1BLH** [reply]
@phl0 
Log-file always sounds good to find solution. I set the log_treshold variable to 4, but threre no new messages in the log-file after try to connect from gridtracker.  Only routine messages are in the log-file -- when i refresh page of the cloudlog).
If i would set log_treshold variable to 1 (only error logging) -- log file is absent.
I have no idea how to find log-file of the gridtracker. :(

P.S. the strange thing is tha all worked till blackout. I tryed to update files from git, but it gave no result.

**Reply by phl0** [reply]
That sounds like gridtracker has an issue. Maybe DNS problem?!

**Reply by R1BLH** [reply]
> That sounds like gridtracker has an issue. Maybe DNS problem?!

Looks like something happened in my firewall/router configuration. Thanks for the idea.
Both the gridtracker PC and the cloudlog server are on my local network.
I tried to connect gridtracker via vpn with an endpoint in the Netherlands and that connection was successful. 
However, I can connect to cloudlog from the gridtracker PC via a browser without vpn.
This is a really strange situation after blackout :)

**Reply by R1BLH** [reply]
> Maybe DNS problem?!

There is a problem with IPv6 firewall. Ping is OK, but server resources are unreachable over IPv6. 
Thank you again for idea. :)
